validator/src/test/java/io/smallrye/config/validator/ValidateConfigTest.java

```diff
@@ -124,7 +123,7 @@ void validateConfigMapping() {
                 "server.info.admins.root[1].username size must be between 0 and 4",
                 "server.info.admins.root size must be between 0 and 1",
                 "server.info.firewall.accepted[1] size must be between 8 and 15",
-                "server is not prod");
+                "server server is not prod");
```

Comment:
Add more levels of nesting to ensure the path is being built correctly.

Reply:
I added a class level validation that has more nesting elsewhere in the test. You can find it a few lines above this one. The test related to this validation is:
```"server.cors.origins[2] someClassLevelCrossValidation If host is server3, then port value must be 3",```

Reply:
It is enough.

What I don't understand is the double `server` in the message. Shouldn't we just use the message as is from the custom annotation?

Reply:
In the test, the first "server" is the full path to the failing object. Then the following string "server is not prod" is the error message defined for this validation.

The result is weird but the algorithm works as intended.